pkg/process/checks/container.go

```diff
@@ -160,11 +160,11 @@ func fmtContainers(ctrList []*containers.Container, lastRates map[string]util.Co
 		containers = append(containers, &model.Container{
 			Id:          ctr.ID,
 			Type:        ctr.Type,
-			CpuLimit:    float32(ctr.CPULimit),
+			CpuLimit:    float32(ctr.Limits.CPULimit),
```

Comment:
@DataDog/processes Currently CPU Limit is computed differently whether containers have limits or not.
Let's take an example:
- We have a 4 CPUs node
- We have 1 container running without limit (containerA)
- We have 1 container running with a limit set to 2 CPUs (containerB)

Currently, the `CPULimit` value will be:
- containerA = 100%
- containerB = 200%

While, in reality, `containerA` can use up to 4 CPUs. This PR fixes this and we'll now return `400%` for `containerA`.